components/carousel/utils/useCarouselSpotlight.ts

```diff
@@ -61,12 +61,11 @@ function updateCollections(data) {
 
 export default function useCarouselSpotlight() {
   const collections = useSpotlightNft()
-  const { urlPrefix, client } = usePrefix()
+  const { urlPrefix } = usePrefix()
   const variables = curatedCollection[urlPrefix.value]?.length
     ? { list: curatedCollection[urlPrefix.value] }
     : undefined
   const { data } = useGraphql({
-    clientName: client,
```

Comment:
throw warning without, I think we need to have default value for `clientName` in `useGraphql`

Reply:
It's just a type warning. I will fix it.

Reply:
Still show warning for me:

```
Argument of type '{ queryName: string; queryPrefix: Prefix; variables: { list: any; } | undefined; }' is not assignable to parameter of type '{ queryPrefix?: string | undefined; queryName: any; clientName: any; variables?: {} | undefined; options?: {} | undefined; disabled?: ComputedRef<boolean> | undefined; data?: Ref<...> | undefined; error?: Ref<...> | undefined; loading?: Ref<...> | undefined; }'.
  Property 'clientName' is missing in type '{ queryName: string; queryPrefix: Prefix; variables: { list: any; } | undefined; }' but required in type '{ queryPrefix?: string | undefined; queryName: any; clientName: any; variables?: {} | undefined; options?: {} | undefined; disabled?: ComputedRef<boolean> | undefined; data?: Ref<...> | undefined; error?: Ref<...> | undefined; loading?: Ref<...> | undefined; }'.
```

Reply:
works with https://github.com/kodadot/nft-gallery/pull/7208#discussion_r1319546597 this change

Reply:
thank you for your update